AGENTS.md

```
@@ -327,12 +327,14 @@ Backend spans `app/` + `core/` (unified API + domain logic).
 
 ### Update flow
 1. From repo root: `make openapi` (generates OpenAPI + regenerates TS types).
-2. Commit changes to:
-   - `frontend/src/api/openapi.json`
-   - `frontend/src/api/schema.ts`
+2. **Mandatory:** Commit changes to:
+ - `frontend/src/api/openapi.json`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)